chandra_aca/aca_image.py

```diff
@@ -309,6 +311,241 @@ def col0(self):
     def col0(self, value):
         self.meta['IMGCOL0'] = np.int64(value)
 
+    @classmethod
+    def _read_flicker_cdfs(cls):
+        """Read flickering pixel model cumulative distribution functions
+        and associated metadata.  Set up class variables accordingly.
+
+        The flicker_cdf file here was created using:
+        /proj/sot/ska/www/ASPECT/ipynb/chandra_aca/flickering-pixel-model.ipynb
+
+        """
+        from astropy.io import fits
```

Comment:
Is this import a heavy one?  Wondering if this could just be at top level.

Reply:
Yes it is a heavy import.  For reasons that make no sense it pulls in a big chunk of scipy (optimize and linalg etc).